src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -333,16 +334,14 @@ public String toString() {
 		printer.write("@");
 		scan(annotation.getAnnotationType());
 		if (annotation.getValues().size() > 0) {
-			printer.write("(");
-			for (Entry<String, CtExpression> e : annotation.getValues().entrySet()) {
-				printer.write(e.getKey() + " = ");
-				elementPrinterHelper.writeAnnotationElement(annotation.getFactory(), e.getValue());
-				printer.write(", ");
+			try (ListPrinter lp = printer.createListPrinter("(", ", ", ")")) {
+				for (Entry<String, CtExpression> e : annotation.getValues().entrySet()) {
+					lp.printSeparatorIfAppropriate();
+					printer.write(e.getKey() + " = ");
+					elementPrinterHelper.writeAnnotationElement(annotation.getFactory(), e.getValue());
+				}
 			}
-			printer.removeLastChar();
-			printer.write(")");
 		}
-		printer.writeln().writeTabs();
```

Comment:
Ok I just saw your comment and the fact that tests are presented only one annotation and not multiple ones, so don't consider my comment ;)